common/changes/office-ui-fabric-react/tabrumle-CustomOverflowLabel_2018-11-26-23-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fix custom title on overflow button not being used in facepile",
```

Comment:
OUIFR has been getting in the habit of prefixing changelog notes with the component name for better release notes. We also need to document this better.

May I suggest modifying this to be:

```json
"comment": "Facepile: Fix custom title on overflow button not being used in Facepile",
```

Reply:
Modified to include your suggestion

Reply:
Perfect, thank you!